drivers/watchdog/wdt_wwdgt_gd32.c

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) 2022 TOKITA Hiroshi
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT gd_gd32_wwdgt
+
+#include <zephyr/drivers/watchdog.h>
+
+#include <gd32_wwdgt.h>
+#include <soc.h>
+
+#define LOG_LEVEL CONFIG_WDT_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(wdt_wwdgt_gd32);
+
+#define WWDGT_NODE DT_NODELABEL(wwdgt)
```

Comment:
why is this needed? you have `DT_INST*` available

Reply:
Use DT_INST_PROP(0, ...)